src/Task_21/Main.java

```diff
@@ -0,0 +1,21 @@
+package Task_21;
+
+import java.util.Scanner;
+import java.util.TreeSet;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        Scanner scanner = new Scanner(System.in);
+        String filePath;
+        TreeSet<Person> setOfPeople;
+
+//        System.out.println("Введите путь к файлу:");
+//        filePath = scanner.nextLine();
+        filePath = "C:\\Users\\anton\\IdeaProjects\\Person.txt";
+        System.out.println(setOfPeople = PersonService.readFromFile(filePath));
```

Comment:
Please, don't do such inititialization inside the method call. Put `setOfPeople = PersonService.readFromFile(filePath)` in separate line